src/cloud-api-adaptor/install/rbac/peer-pod.yaml

```diff
@@ -71,3 +71,27 @@ roleRef:
   kind: ClusterRole
   name: node-viewer
   apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: pp-secrets
+  namespace: confidential-containers-system
+rules:
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["create", "patch", "update", "get", "watch", "list"]
+---
```

Comment:
Why it is needed? can access be limited to specific secret?
`resourceNames: ["my-configmap"]`

Reply:
The implementation is creating secrets named `pp-<peer pod SID>` under the namespace. These secrets include the key pair each PP. This will be used to regain PP secrets following WN Pod restart.